geem/static/geem/js/geem_validation.js

```diff
@@ -0,0 +1,46 @@
+/**
+ * @file Functions used by validator tab.
+ */
+
+
+/**
+ * Get grid options for validator tab grid.
+ * @returns {Object} Grid options
+ */
+function get_grid_options() {
+	// const form_specs
+	const column_defs = [
+		{headerName: 'Test Column One', field: 'test_column_one'},
+		{headerName: 'Test Column Two', field: 'test_column_two'},
+	];
+
+	return {
+		columnDefs: column_defs,
+		rowData: [],
+	};
+}
+
+
+/**
+ * Create the grid in the validator tab.
+ * @param {Object} grid_options - Grid options
+ */
+function create_grid(grid_options) {
+	const grid_div = document.querySelector('#validation_grid');
+	new agGrid.Grid(grid_div, grid_options);
+}
+
+
+/**
+ * Update the grid with new rows
+ * @param {Object }grid_options - Grid options
+ * @param {string} data - ``csv`` string representation of new grid rows
+ */
+function update_grid(grid_options, data) {
+	let new_rows = data.split('\n');
```

Comment:
Does this assume first row is column labels?  We should demand that.  And enable uploaded data coming into an empty spreadsheet to set the column labels.

Reply:
It does not assume that. I will add it.

Reply:
great

Reply:
Done